src/pages/docs/introduction/installation.mdx

```diff
@@ -17,7 +17,11 @@ The quick install script allows you to easily install the latest version of Jump
 
 ```shell
 curl https://jumppad.dev/install | bash
+```
```

Comment:
As things are, copy and paste copies even the output which is annoying. With this change, it sepapares the output from the command itself